app/policies/admin/admin/github_user_policy.rb

```diff
@@ -1,4 +1,4 @@
-class Admin::GitHubUserPolicy < ApplicationPolicy
```

Comment:
Before you cringe at this, I have to say that I thought about this and this is the "automatic" way of referencing this policy in the same way all other admin policies are loaded. I don't like it either so if you have an idea that would make this better, I'm open to hearing it.